lib/custom_a11y_rules.js

```diff
@@ -156,6 +171,26 @@ var customRules = {
             evaluate: function(node, options) {
                 return axe.utils.isHidden(node);
             },
-        }
+        },
+        {
+            'id': 'section-heading-first-child',
+            'metadata': {
+                'impact': 'minor',
```

Comment:
I see this is the only one that is set with a 'minor' impact. What will that mean when running a bok-choy test?